hypervisor/debug/profiling.c

```diff
@@ -1270,6 +1262,42 @@ int32_t profiling_get_pcpu_id(struct acrn_vm *vm, uint64_t addr)
 	return 0;
 }
 
+/*
+ * Update collection statictics
+ */
+int32_t profiling_get_status_info(struct acrn_vm *vm, uint64_t gpa)
+{
+	uint16_t i;
+	struct profiling_status pstats[phys_cpu_num];
+
+	(void)memset((void *)&pstats, 0U, phys_cpu_num*sizeof(struct profiling_status));
+
+	dev_dbg(ACRN_DBG_PROFILING, "%s: entering", __func__);
+
+	if (copy_from_gpa(vm, &pstats, gpa,
```

Comment:
Why do we need to copy these data from vm to HV? They are not used in HV actually.

Reply:
Thanks for catching. You don't need it. I submit the revised patch.